README.md

```diff
@@ -61,30 +61,36 @@ backend.when('GET', '/api/users/*').respond({
 });
 ```
 
-Then use XHR per usual:
+Then use XHR or any AJAX library you wish to do your tests (jQuery with [chai](http://chaijs.com/) should assertions below):
 
 ```javascript
-var xhr = new XMLHttpRequest();
-
-xhr.onreadystatechange = function () {
-  if (this.readyState === 4 && this.status === 200) {
-    var data = JSON.parse(xhr.responseText);
-    var msg = '<p>Hello, ' + data.name + ' the ' + data.species + '</p>';
-
-    document.body.innerHTML += msg;
-  }
-};
-
-xhr.open('GET', '/api/users/jake');
-xhr.send();
```

Comment:
You had more documentation on how to create an ajax request then actually using this library. the readme should focus on how to use the library not how to do something the already know how to.